hotel/data/hotel_data.xml

```diff
@@ -0,0 +1,169 @@
+<odoo noupdate="0">
```

Comment:
could this file go in a `demo` directory`? It would be clearer according to me.

Reply:
Done